apps/data/database/initdb.sql

```diff
@@ -0,0 +1,93 @@
+CREATE FUNCTION trigger_update_modified()
```

Comment:
I may be misunderstanding something, but I'm not sure why there are sql commands here. Is this for testing?